include/nbl/builtin/hlsl/cpp_compat.hlsl

```diff
@@ -6,6 +6,7 @@
 #include <type_traits>
 
 #define ARROW ->
+#define NBL_CONST const
```

Comment:
@devshgraphicsprogramming approved of this?
cause I know you did it for const functions but it can be ambigous and confusing